code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -97,6 +97,77 @@
 	name = "trophy rack"
 	desc = "It's useful for both carrying extra gear and proudly declaring your insanity."
 	icon_state = "cultpack"
+	var/obj/item/organ/external/head/LS
+	var/obj/item/organ/external/head/MS
+	var/obj/item/organ/external/head/RS
+	var/mutable_appearance/LSma
+	var/mutable_appearance/MSma
+	var/mutable_appearance/RSma
+	var/placement
+
+/obj/item/storage/backpack/cultpack/attackby(obj/item/I, mob/living/user)
```

Comment:
This would all probably be better as an `update_icon` proc that just loops through the contents and fetches the appearance of the first three heads and places them at specific overlayed locations, with the 4th and on head just ignored.

Then just ensure the icon updates when something is taken out or put in.

Having an input proc like this is kinda ick and make using the pack with limbs clunky.

Reply:
Proposed solution would remove about 95% of the flavour of the PR. How often are chaplains sticking heads in their backpacks during combat that it requires an ultra-streamlined workflow?